### PR TITLE
Add queue admin operations support

### PR DESCRIPTION
**Pull request by erandacr**
## Purpose
This will add queue admin operations support, which will include queue info retrieval, queue creation and queue deletion.

Resolves https://github.com/ballerina-platform/ballerina-message-broker/issues/261

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: modules/broker-cli-client/src/main/java/io/ballerina/messaging/broker/client/cmd/impl/create/CreateQueueCmd.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    @Override
+    public void execute() {
+        if (help) {
+            processHelpLogs();
+            return;
+        }
+
+        Configuration configuration = Utils.readConfigurationFile();
+        HttpClient httpClient = new HttpClient(configuration);
+        String urlSuffix = "queues/";
```
Comment (a5anka) [inline modules/broker-cli-client/src/main/java/io/ballerina/messaging/broker/client/cmd/impl/create/CreateQueueCmd.java]: Shouldn't this be a class constant?

Comment (erandacr) [reply modules/broker-cli-client/src/main/java/io/ballerina/messaging/broker/client/cmd/impl/create/CreateQueueCmd.java]: +1

File: modules/broker-cli-client/src/main/java/io/ballerina/messaging/broker/client/http/HttpClient.java [hunk 2]
```diff
@@ -95,24 +96,14 @@ public HttpResponse sendHttpRequest(HttpRequest request, String httpMethod) {
             con.setRequestProperty("Accept", "application/json");
             con.setRequestProperty("Content-Type", "application/json");
             con.setRequestProperty("Authorization", "Basic" + encodedCredentials);
+            // Disable kee-alive. Since client will send one request per each startup, it won't get any advantage
```
Comment (a5anka) [inline modules/broker-cli-client/src/main/java/io/ballerina/messaging/broker/client/http/HttpClient.java]: I think kee-alive should be keep-alive

File: modules/broker-cli-client/src/main/java/io/ballerina/messaging/broker/client/http/HttpClient.java [hunk 3]
```diff
@@ -121,18 +112,18 @@ public HttpResponse sendHttpRequest(HttpRequest request, String httpMethod) {
             BufferedReader in = null;
             StringBuilder response = new StringBuilder();
             try {
-                // if an error code is returned retrieve the errorStream otherwise inputStream
-                if (responseCode / 100 == 4 || responseCode / 100 == 5) {
+                // if  non-error code is returned retrieve the inputStream otherwise errorStream
```
Comment (a5anka) [inline modules/broker-cli-client/src/main/java/io/ballerina/messaging/broker/client/http/HttpClient.java]: Two spaces?

File: modules/broker-cli-client/src/main/java/io/ballerina/messaging/broker/client/http/HttpClient.java (hunk 2 above)
Comment (erandacr) [reply modules/broker-cli-client/src/main/java/io/ballerina/messaging/broker/client/http/HttpClient.java]: will fix :)

File: modules/broker-cli-client/src/main/java/io/ballerina/messaging/broker/client/http/HttpClient.java (hunk 3 above)
Comment (erandacr) [reply modules/broker-cli-client/src/main/java/io/ballerina/messaging/broker/client/http/HttpClient.java]: yes, will fix. thanks

**Pull request merged**